### PR TITLE
logging unhandled promise rejection

### LINKED ISSUE #5654: Stress Test - top-level listener for uncaught exceptions/rejections
## Work Item
**Describe the outcome you expect**
Uncaught exceptions and Promise rejections should appear in the logs for the stress test.

**Approach**
[uncaughtExceptionMonitor](https://nodejs.org/api/process.html#process_event_uncaughtexceptionmonitor) looks like what we want, and just log an error using the logger when it happens.

**Open questions**
We should also think through what to do with the runner process itself in these cases - let it crash and the orchestrator restarts it? Catch the exception and "reset" the runner to keep going?

**Additional context**
Breaking this out as a work item for #5304

### PR DESCRIPTION
**Pull request by chensixx**
Fixes #5654

### CODE REVIEW

**Review by anthony-murphy — APPROVED** [review]

File: packages/test/test-service-load/src/runner.ts [hunk 1]
```diff
@@ -68,6 +68,14 @@ async function main() {
     // will get its own set of randoms
     randEng.seedWithArray([seed, runId]);
 
+    const l = await loggerP;
+    process.on("unhandledRejection", (reason, promise) => {
+        try{
+            l.sendErrorEvent({eventName: "UnhandeldPromiseRejection"}, reason);
```
Comment (markfields) [inline packages/test/test-service-load/src/runner.ts:74]:
```suggestion
            l.sendErrorEvent({eventName: "UnhandledPromiseRejection"}, reason);
```

File: packages/test/test-service-load/src/runner.ts [hunk 2]
```diff
@@ -68,6 +68,14 @@ async function main() {
...
+        } catch(e) {
+            console.log("Error during logging unhandled promise rejection: ", e);
```
Comment (markfields) [inline packages/test/test-service-load/src/runner.ts:76]:
```suggestion
            console.error("Error during logging unhandled promise rejection: ", e);
```

to make it stand out more, not that it's ever going to happen...

**Review by markfields — APPROVED** [review]
🎂 LGTM except for a typo which I made a suggestion to fix :)

**Review by markfields — APPROVED** [review]
🎂

**Pull request merged**